tools/who_what_benchmark/whowhatbench/wwb.py

```diff
@@ -147,11 +156,60 @@ def load_text2image_model(
     return model
 
 
+def load_visual_text_genai_pipeline(model_dir, device="CPU", ov_config=None):
+    try:
+        import openvino_genai
+    except ImportError:
+        logger.error("Failed to import openvino_genai package. Please install it.")
```

Comment:
does it make sense to print original error message too here?

Reply:
why not, added